R/latex.R

```diff
@@ -158,13 +158,12 @@ latexmk_emu = function(
 ) {
   aux = c(
     'log', 'idx', 'aux', 'bcf', 'blg', 'bbl', 'fls', 'out', 'lof', 'lot', 'toc',
-    'nav', 'snm', 'vrb', 'ilg', 'ind', 'xwm', 'brf', 'run.xml'
+    'nav', 'snm', 'vrb', 'ilg', 'ind', 'xwm', 'brf', 'run.xml', 'preview.aux'
```

Comment:
`preview.aux` is a filename instead of a suffix, so it can't be placed inside the `aux` vector, otherwise the aux file to be cleaned up would be `foo.preview.aux` instead of `preview.aux` when the input file is `foo.tex`.

Reply:
Ok, thanks for the clarification.
The one thing I don't understand is, when I just changed the line of code that accesses `aux_files['log']` back to `aux_files[1]`, which then deleted the file, it fixed the error I was getting. It isn't clear why a file deletion was causing issues with `output_dir` being specified.

Reply:
I don't have an explanation without further investigation, but could you verify if my revision of the PR actually works?

```r
remotes::install_github('yihui/tinytex')
```

Thanks!

Reply:
Yes, doing now.

Reply:
It appears to work. I did the following:

- Installed `tinytex` using the command you gave
- confirmed after install that `tinytex:::latexmk_emu` had your code change.
- I ran the test in my post

This ran w/o error and produced the expected PDF (which I was able to open w/o any issue).

-Steve

Reply:
Great. Thanks a lot!

Reply:
@yihui quick question, do you have an ETA on the next release ? I am just planning around when that will happen. Thanks again for you help.

Reply:
No problem. I can do it this week.

Reply:
The new release is on CRAN now.

Reply:
@yihui FYI, I updated to 0.31 (currently available using source), and things appear to work locally (this is expected, but just confirming).